framework/source/class/qx/bom/client/EcmaScript.js

```diff
@@ -117,6 +117,26 @@ qx.Bootstrap.define("qx.bom.client.EcmaScript",
 
 
     /**
+     * Checks if 'find' is supported on the Array object.
+     * @internal
+     * @return {Boolean} <code>true</code>, if the method is available.
+     */
+    getArrayFind : function() {
```

Comment:
I'd agree except that it follows the convention of all the other Array feature detection methods in that class of "getArray" + the name of the method, eg getArrayReduce, getArrayEvery, getArraySome, etc